docs/continuous-integration.md

```diff
@@ -41,8 +41,7 @@ These scripts will all:
 3. Compare the two reports
 4. If allowed to, post back a status to the PR (the HEAD commit)
 
-These scripts will detect your npm client (yarn or npm) and run `npm
-install`/`yarn install` before running happo on the commits. If you have other
+These scripts will detect your npm client (yarn or npm) and run `npm install`/`yarn install` before running happo on the commits. If you have other
```

Comment:
Odd -- I expected this to wrap.

Reply:
I've now set `proseWrap` to `always` and ran Prettier again. Do you like the new style?

Quote from the Prettier docs: [^prettier]

> By default, Prettier will not change wrapping in markdown text since some services use a linebreak-sensitive renderer, e.g. GitHub comments and BitBucket. To have Prettier wrap prose to the print width, change this option to `"always"`. If you want Prettier to force all prose blocks to be on a single line and rely on editor/viewer soft wrapping instead, you can use `"never"`.

[^prettier]: https://prettier.io/docs/en/options.html#prose-wrap

Reply:
Yep, that looks better!